operator/src/main/java/oracle/kubernetes/operator/steps/WatchDomainIntrospectorJobReadyStep.java

```diff
@@ -33,15 +39,22 @@ public NextAction apply(Packet packet) {
 
     // No need to spawn a watcher if the job is already complete
     if (domainIntrospectorJob != null && !JobWatcher.isComplete(domainIntrospectorJob)) {
-      JobWatcher jw =
-          JobWatcher.create(
-              ThreadFactorySingleton.getInstance(),
-              namespace,
-              initialResourceVersion,
-              tuning,
-              new AtomicBoolean(false));
-
-      return doNext(jw.waitForReady(domainIntrospectorJob, getNext()), packet);
+      JobWatcher jw = null;
+      if (jws == null || !jws.containsKey(namespace)) {
+        jw =
+            JobWatcher.create(
+                ThreadFactorySingleton.getInstance(),
+                namespace,
+                initialResourceVersion,
+                tuning,
+                isStopping != null ? isStopping : new AtomicBoolean(false));
```

Comment:
isStopping must not be null, so I don't want to pass in a new AtomicBoolean(false)

Reply:
ummm. ok. so if the caller passes in a null at JobHelper.createDomainIntrospectorJobStep(), how do we want to handle it there? should we put a default in place there or throw an exception?

Reply:
I think these will help. Let me know.